{"content": {"suites": {"0": {"status": {"total_pass": 3, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_all_project_info", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_project_info_models", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_project_users_models", "rerun": "0"}, "3": {"status": "FAIL", "message": "       django.core.exceptions.SuspiciousFileOperation: The joined path (D:\\get\\such\\laugh.xlsx) is located outside of the base path component (D:\\projects\\backend\\backend\\static\\media_files)\n", "test_name": "test_file_upload", "rerun": "0"}}, "suite_name": "project/Test_project/test_models.py"}}}, "date": "August 30, 2021", "start_time": 1630325302.7951365, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "3", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "4"}